src/dialects/abstract/query-generator-typescript.ts

```diff
@@ -10,6 +10,15 @@ import type { AbstractDialect } from './index.js';
 
 export type TableNameOrModel = TableName | ModelStatic;
 
+// keep REMOVE_INDEX_QUERY_SUPPORTABLE_OPTIONS updated when modifying this
+export type RemoveIndexQueryOptions = {
+  concurrently?: boolean,
+  ifExists?: boolean,
+  cascade?: boolean,
+};
+
+export const REMOVE_INDEX_QUERY_SUPPORTABLE_OPTIONS = new Set(['concurrently', 'ifExists', 'cascade']);
```

Comment:
I was wondering if there was a nice way where we only had to update the type and the Set would automatically update as well

Reply:
It would save some potential typo's if this were possible.
From my limited knowledge this isn't possible as the set is an actual value whereas the type/interface is just a type; however, I'm open to comments.

Reply:
I've added this in, it works quite well.
LMK if you have any comments on it